packages/contracts-bedrock/scripts/deploy/DeployOPPrestateUpdater.s.sol

```diff
@@ -8,331 +8,96 @@ import { Script } from "forge-std/Script.sol";
+        (blueprints.permissionlessDisputeGame1, blueprints.permissionlessDisputeGame2) = DeployUtils.createDeterministicBlueprint(faultDisputeGameInitcode, _salt);
```

Comment:
nit, let's add assertions that `permissionlessDisputeGame2` is not the zero address, since we know this is big enough that it needs two blueprints, as a sanity check we have the right initcode

Reply:
Done, and also now printing the initCodeHash along with a link to the 1.8.0 initCodeHash:

bfffb469d1e5